library/kani/src/arbitrary.rs

```diff
@@ -13,6 +13,7 @@ pub trait Arbitrary {
 impl<T> Arbitrary for T
 where
     T: Invariant,
+    [(); std::mem::size_of::<T>()]:,
```

Comment:
Add an explanatory comment here as well? SImilar to the comment in `any_raw`.

Reply:
Done